src/lib/styles/elements/breadcrumbs.css

```diff
@@ -0,0 +1,13 @@
+/* Tailwind Elements: breadcrumbs.css */
+
+@layer components {
+	.breadcrumbs {
+		@apply flex items-center space-x-4 w-full hide-scrollbar overflow-x-auto;
+	}
+	.crumb {
+		@apply flex justify-center items-center space-x-2;
+	}
+	.crumb-separator {
```

Comment:
Likewise, but in order to support third party icon libraries folks need to be able to define the symbol using HTML markup. If we did it purely through CSS this would not be possible.

I've made the change to the docs and our own implementation of the crumb elements. Good call!